### ISSUE TITLE
Common: add encrypted S3 credentials to .travis.yml

### ISSUE THREAD

**Issue by fblundun** [snowplow/snowplow#2673]
The variables should be called:

* `AWS_DEPLOY_ACCESS_KEY_ID=xxx`
* `AWS_DEPLOY_SECRET_ACCESS_KEY=xxx`

The `_DEPLOY` slug is in case we add other AWS creds in the future (e.g. for integration testing).

Add with `travis encrypt XXX=xxx --add`

**Comment by alexanderdean** [comment]
Re-assigning to @fblundun as he has all the info required now...

**Comment by alexanderdean** [comment]
Let's tweak the name of this ticket, it's a bit wordy

**Comment by fblundun** [comment]
How about "Common: add encrypted S3 credentials to .travis.yml"?

**Comment by alexanderdean** [comment]
Sounds good!
